www/main/navigator/shabad/ShabadHeader.jsx

```diff
@@ -46,11 +46,37 @@ const ShabadHeader = () => {
 
   return (
     <div className="shabad-pane-header">
+      {activeShabadId && (
+        <button
+          className={favShabadIndex < 0 ? 'button fav-btn' : 'button fav-btn unfav-btn'}
+          ref={favBtnRef}
+					title={i18n.t('SHABAD_PANE.FAV_BTN_TOOLTIP')}
+          onClick={() => {
+            if (favShabadIndex < 0) {
+							const timestamp = new Date();
+              const shabadObj = {
+                text: searchVerse,
+                shabadId: activeShabadId,
+                verseId: activeVerseId,
+								timestamp
+              };
```

Comment:
Veerji I think this piece of code needs some formatting done.

PS you can use that `classNames` function here as well.

Reply:
Should be fixed now, have a look